queries/omen/markets.ts

```diff
@@ -332,6 +393,26 @@ const getMarketUserTrades = async (params: QueryFpmmTradesArgs & FpmmTrade_Filte
 const getMarketTrades = async (params: QueryFpmmTradesArgs & FpmmTrade_Filter) =>
   request<Pick<Query, 'fpmmTrades'>>(OMEN_SUBGRAPH_URL, getMarketTradesQuery, params);
 
+const getMarketTradesAndTransactions = async (
+  params: QueryFpmmTradesArgs & QueryFpmmTransactionsArgs & FpmmTrade_Filter
+) => {
+  const modifiedParams = {
+    ...params,
+    tradesFirst: params.first || 0,
+    // we need to get 2 more trasanction than trade
+    // to handle liquidity tx (add/remove) that have no associated trade
+    // by getting 2 more tx we match all trades
+    // even if there is 2 liquidity tx in there
```

Comment:
Can we simplify this message? Something like "Txs can have +2 results than trades. We need to fetch +2 to be able to merge tx and trades"